src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/VaccinationProductType.swift

```diff
@@ -0,0 +1,27 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+enum VaccinationProductType {
+	case biontech
+	case moderna
+	case astraZeneca
+	case unknown
```

Comment:
not sure if compiler allows this but:
```suggestion
	case other
```
I think this would make it easier to understand the guard let in `HealthCertifiedPerson.swift`: `VaccinationProductType(value: vaccinationEntry.vaccineMedicinalProduct) != .unknown else`

Reply:
Done